example.conf

```diff
@@ -220,4 +226,55 @@ GUBER_ADVERTISE_ADDRESS=localhost:9990
 # Choose the number of replications
 # GUBER_REPLICATED_HASH_REPLICAS=512
 
+############################
+# OTEL Tracing Config
+# See
+############################
+
+# Set the name of the service which will be reported in traces
+# OTEL_SERVICE_NAME=gubernator
+
+# Set the tracing level, this controls the number of spans included in a single trace.
+#    Valid options are (ERROR, INFO, DEBUG) Defaults to "ERROR"
+# GUBER_TRACING_LEVEL=ERROR
+
+# Set which sampler to use (always_on, always_off, traceidratio, parentbased_always_on,
+#    parentbased_always_off, parentbased_traceidratio)
+# OTEL_TRACES_SAMPLER=always_on
+
+# If traceidratio set the ratio to a value between 0.0 and 1.0
+# OTEL_TRACES_SAMPLER_ARG=1.0
+
+# Choose the transport protocol (otlp, grpc or http/protobuf)
+# OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+
+# The endpoint to send traces to
+# OTEL_EXPORTER_OTLP_ENDPOINT=https://collector:<port>
+
+# If the endpoint is `https` specify the TLS certificate files
+# OTEL_EXPORTER_OTLP_CERTIFICATE=/path/to/cert
+# OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE=/path/to/client
+# OTEL_EXPORTER_OTLP_CLIENT_KEY=/path/to/ca
+
+# Optional headers appended to the http requests in the format key=value,key2=value2
+# OTEL_EXPORTER_OTLP_HEADERS=header1=value1
+
+############################
+# Honeycomb.io Tracing Config
+############################
+# OTEL_EXPORTER_OTLP_PROTOCOL=otlp
+# OTEL_EXPORTER_OTLP_ENDPOINT=https://api.honeycomb.io
+# OTEL_EXPORTER_OTLP_HEADERS=x-honeycomb-team=<api-key>
+
+############################
```

Comment:
Probably ok to remove Jaeger config example because OTel has deprecated the protocol.  However, you can still send traces to Jaeger server via OTLP protocol.  That might be a better example.